sprout/requirements.txt

```diff
@@ -1,5 +1,5 @@
 Command
-Django<1.10
+Django>1.10.7,<1.11
 -e git+git://github.com/izapolsk/django-json-field.git#egg=json_field
```

Comment:
btw pip install warns about this, since `egg` metadata doesn't match django-json-field.  Might want comment in-line for why the egg metadata is changed here.